pkg/smart_wallet/config.go

```diff
@@ -0,0 +1,94 @@
+package smart_wallet
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// SmartWalletConfig represents the configuration
+// for the smart wallet to be used with the client.
+type Config struct {
```

Comment:
Add `SMART_WALLET_` prefix to all env names

Reply:
3c951ab39e20cf6645d24e3e86bde69c2c0460c6